### ISSUE TITLE
Update Kubernetes dependency to 1.18

### ISSUE THREAD

**Issue by turkenh** [vmware-tanzu/velero#2540]
**What steps did you take and what happened:**
Getting a lot of errors similar to following when I want to build my project, which depends velero, with k8s 1.18:

```
pkg/generated/clientset/versioned/typed/velero/v1/backup.go:74:5: not enough arguments in call to c.client.Get().Namespace(c.ns).Resource("backups").Name(name).VersionedParams(&options, scheme.ParameterCodec).Do
	have ()
	want (context.Context)
```

This is because velero depends on k8s 1.17 and clientset generated with that version.

**What did you expect to happen:**
I can consume velero together with latest Kubernetes versions.

**Comment by deepakkinni** [comment]
Is there an ETA for this?

**Comment by laverya** [comment]
I've opened a PR to update to k8s 1.18 - only major problem I foresee is that the external-snapshotter import is now a prerelease version (since the currently released version is still k8s 1.17)
